### ISSUE TITLE
Menu is incorrectly scaled compared to remaining UI on HiDPI display

### ISSUE THREAD

**Issue by b52** [atom/atom#13331]
### Description

Running latest Atom (v1.12.5) the menu is basically doubled in size compared to the rest of the UI.

![2016-11-26-113126_1930x1323_scrot](https://cloud.githubusercontent.com/assets/23704/20639734/fb68ce92-b3cb-11e6-913c-2f9180860cf0.png)

My previous version (v1.11.2) didn't have this issue. I start Atom without any special options (the screenshot was taking using `--safe`). However, I use the following UI related options to scale UIs properly for my HiDPI display.

**.xinitrc**
```
export QT_SCREEN_SCALE_FACTORS=2
export GDK_SCALE=2
export GDK_DPI_SCALE=0.5
```

**.Xresources**
```
Xft.dpi: 192
```

### Steps to Reproduce

1. Have a HiDPI display.
2. Use the above environment variables to scale all UIs properly.
3. Start Atom without any special option.

**Expected behavior:**

A normally sized Atom, including the menu.

**Actual behavior:**

A normally sized Atom, excluding the menu.

### Versions

```
Archlinux

Atom    : 1.12.5
Electron: 1.4.8
Chrome  : 53.0.2785.143
Node    : 6.5.0

apm  1.14.1
npm  4.0.2
node 7.2.0
python 3.5.2
git 2.10.2
```

**Comment by thomasjo** [comment]
Thanks for reaching out!

We've determined that this issue is being caused by the use of an unofficial build or distribution of Atom. Often these customized versions of Atom are modified versions of the Stable branch of Atom with mismatched versions of built-in components. These updated components are taken from the Beta channel or `master` branch and then injected into the Stable version and a new Atom package is generated. Because of the way Atom is constructed, using these mismatched components can cause mysterious and hard-to-diagnose problems. You can find out more about [why we chose to not support unofficial distributions here](https://github.com/atom/design-decisions/blob/master/unofficial-atom-distributions.md).

---
**TL;DR** Electron v1.4.8 has been incorrectly patched into Atom by your AUR distribution of Atom. Use another AUR package, or build from source.

**Comment by b52** [comment]
Thanks for the hint. I replied to a [correspondig bug ticket](https://bugs.archlinux.org/task/50357?string=atom&project=5&search_name=&type[0]=&sev[0]=&pri[0]=&due[0]=&reported[0]=&cat[0]=&status[0]=open&percent[0]=&opened=&dev=&closed=&duedatefrom=&duedateto=&changedfrom=&changedto=&openedfrom=&openedto=&closedfrom=&closedto=) in the Arch bug tracker, which describes exactly this issue. I hope they get convinced to bundle Electron as required.
